practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>This should be in an initial cell so it can all be run together before the Indaba.</p><p> </p><p>Given that the code is missing, I'm going to assume that the prac is not yet finished. Please ping me on slack or via email when it is and I will review it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yep indeed. I will ping you once done. Thanks @mmorris44

Reply:
Hi @mmorris44 you can review now the GNN section.